pkg/utils/host/host.go

```diff
@@ -62,6 +65,27 @@ func init() {
 	}
 
 	IsHostPidNs = selfStat.Ino == systemdStat.Ino
+
+	// Initialize HostNumCPU
+	out, _ := exec.Command("lscpu").Output()
+	outstring := strings.TrimSpace(string(out))
+	lines := strings.Split(outstring, "\n")
+	for _, line := range lines {
+		fields := strings.Split(line, ":")
+		if len(fields) != 2 {
+			continue
+		}
+		key := strings.TrimSpace(fields[0])
+		value := strings.TrimSpace(fields[1])
+
+		if key == "CPU(s)" {
+			HostNumCPU, err = strconv.Atoi(value)
+			if err != nil {
+				HostNumCPU = 0
+			}
+			break
+		}
+	}
 }
```

Comment:
Some comments:
- I don't think parsing the output of `lscpu` is the right approach. That command couldn't be available in some environments, so it's better to get this information directly from golang.
- We need to take the "online cpus" number. This can change at runtime, so we'll need to calculate it each time instead of caching it. 

We can use https://pkg.go.dev/github.com/tklauser/numcpus#GetOnline instead of having to parse any file manually.